test.txt

```diff
@@ -0,0 +1 @@
+Here comes the sun!
```

Comment:
Hm... I thought that I removed this file... It was trial with my repository. I have quite little understanding of  how it works. I'm new to it.

Do you have comments about other changes?